Civi/Token/TokenCompatSubscriber.php

```diff
@@ -149,7 +149,6 @@ public function onRender(TokenRenderEvent $e) {
 
     if (!empty($e->context['contact'])) {
       \CRM_Utils_Token::replaceGreetingTokens($e->string, $e->context['contact'], $e->context['contact']['contact_id'], NULL, $useSmarty);
-      $e->string = \CRM_Utils_Token::replaceContactTokens($e->string, $e->context['contact'], $isHtml, $e->message['tokens'], TRUE, $useSmarty);
```

Comment:
The only difference between how this is called here & how it is called in `replaceGreetingTokens` is that in the former case it always sets TRUE for `$isHtml` - however it wouldn't matter because the tokens are replaced in the first function so true vs false is meaningless here.